assets/app/scripts/controllers/projects.js

```diff
@@ -19,9 +19,11 @@ angular.module('openshiftConsole')
     });
     AlertMessageService.clearAlerts();
 
-    $('#openshift-logo').on('click.projectsPage', function() {
-      // Force a reload. Angular doesn't reload the view when the URL doesn't change.
-      $route.reload();
+    $timeout(function() {
```

Comment:
We should probably avoid `$timeout` hacks like this.  We use them when we are doing work in a controller that should really be in a directive.